saleor/graphql/account/mutations.py

```diff
@@ -148,6 +152,32 @@ class Meta:
         model = models.User
 
 
+class LoggedCustomerUpdate(CustomerCreate):
+    class Arguments:
+        id = graphene.ID(
+            description='ID of a customer to update.', required=True)
+        input = UserAddressInput(
+            description='Fields required to update logged in customer.',
+            required=True)
+
+    class Meta:
+        description = 'Updates an existing logged in customer.'
```

Comment:
Not very helpful tho. It allows logged user (not only customer) to update his own data.